exceptions/does_not_exist.py

```diff
@@ -232,6 +232,29 @@ def ROLE_NAME(cls) -> str:  # noqa: N802,N805
         """The name of the Discord role that does not exist."""  # noqa: D401
         return "Archivist"
 
+class ApplicantRoleDoesNotExistError(RoleDoesNotExistError):
+    """Exception class to raise when the "Applicant" Discord role is missing."""
+
+    @classproperty
+    def ERROR_CODE(cls) -> str:  # noqa: N802, N805
+        """The unique error code for users to tell admins about an error that occured."""  # noqa: D401
+        return "E1025"
+
+    @classproperty
+    def DEPENDENT_COMMANDS(cls) -> frozenset[str]: # noqa: N802, N805
+        """
+        The set of names of bot commands that require this Discord entity.
+
+        This set being empty could mean thta all bot commands require this entity,
+        or that none of them do.
+        """  # noqa: D401
+        return frozenset({"make_applicant", "user_make_applicant", "message_make_applicant"})
```

Comment:
Missing additional commands that will rely on the "Applicant" role, once they have been implemented to use the bot's shortcut accessor rather than dynamic access for every command callback. E.g. add `induct` command as a dependent.

Reply:
induct doesn't *depend* on the applicant role though - the induct command will work exactly as intended if the applicant role doesn't exist

Reply:
Ah very true. I suppose the change should be made in the induct cog to still use the shortcut accessor, but use ` with contextlib.suppress(ApplicantRoleDoesNotExistError)`, so that it continues execution.